### ISSUE TITLE
UI : the vertical toolbar is too low : icons are hidden on small screens

### ISSUE THREAD

**Issue by MaelREBOUX** [georchestra/mapstore2-cadastrapp#125]
Here in Rennes we have many users that have new screens with a 16:10 ratio with relatively small dimensions : 1600 x 900 px.
Result : the help icon of the vertical toolbar is hidden and we cannot clic on it.

We propose 4 very small CSS modifications to enhance that.

Modification in Cadastrap CSS :  https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/cadastrapp.css#L38-L43

```css
.cadastrapp .side-bar {
  width: 46px;
  margin-top: 0;
  height: 100%;
  background: #eee;
}
```
And well : I cannot find which file have to be modified in Mapstore2-georchestra (default.css) ?!
Below the CSS I modified in the developer console of my browser :

```css
.geOrchestra .square-button {
	height: 46px;
	width: 46px;
	outline: 0;
}
```

in summary : 52 px -> 46 px + width 46 px +  margin-top: 0

Before    /    After

![capture_252](https://user-images.githubusercontent.com/6370443/120344260-b8eb3680-c2f9-11eb-96fd-ec971874feaa.png)
    
![capture_251](https://user-images.githubusercontent.com/6370443/120344287-bee11780-c2f9-11eb-8942-3225c3b99231.png)

**Comment by landryb** [comment]
heh, fun one.. similar to georchestra/mapstore2-georchestra#385 :)

**Comment by MaelREBOUX** [comment]
1650x1050 screens should be banned...

**Comment by MaelREBOUX** [comment]
Modification in Cadastrap CSS :  https://github.com/georchestra/mapstore2-cadastrapp/blob/master/js/extension/cadastrapp.css#L38-L43

```css
.cadastrapp .side-bar {
  width: 46px;
  margin-top: 0;
  height: 100%;
  background: #eee;
}
```
And well : I cannot find which file have to be modified in Mapstore2-georchestra (default.css) ?!
Below the CSS I modified in the developer console of my browser :

```css
.geOrchestra .square-button {
	height: 46px;
	width: 46px;
	outline: 0;
}
```

in summary : 52 px -> 46 px + width 46 px +  margin-top: 0

**Comment by MaelREBOUX** [comment]
tested and approved with the rc12 version  on 1600x900 screen

![image](https://user-images.githubusercontent.com/6370443/120815637-4cb64000-c550-11eb-91a3-54d96df9b624.png)

**Comment by MaelREBOUX** [comment]
thanks @offtherailz

**Comment by landryb** [comment]
tested too, indeed improved layout on small screens. thanks !
